apps/stripe/src/modules/stripe/errors.ts

```diff
@@ -0,0 +1,13 @@
+import { BaseError } from "@/lib/errors";
+
+export const CreatePaymentIntentError = BaseError.subclass("CreatePaymentIntentError", {
```

Comment:
if you rebase - this file will be removed - I think we can have `StripeEventParsingError` inside its class